anomaly/anomaly_detector.py

```
@@ -243,55 +256,63 @@ def detect_anomaly(data: Dict) -> Tuple[bool, float]:
                 _USING_HEURISTIC_MODE = True
                 health_monitor.mark_degraded(
                     "anomaly_detector",
                     error_msg=f"Model prediction failed: {str(e)}",
-                    fallback_active=True
+                    fallback_active=True,
                 )
                 # Fall through to heuristic
-        
+
         # Use heuristic fallback
         detector_type = "heuristic"
```

Ruff (0.14.10)

Local variable `detector_type` is assigned to but never used

Remove assignment to unused variable `detector_type`

(F841)